src/components/Result.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import cases from 'jest-in-case';
+import Result from './Result';
+
+cases(
+  'should show "I don\'t know" if elements is not an array or if is empty',
+  (opts) => {
+    render(
+      <Result
+        result={{ error: '', expression: {}, elements: opts.elements }}
+      />,
+    );
+    expect(
+      screen.getByText(/I don't know what to say/, { exact: false }),
```

Comment:
Can we just use the `/i` flag for this instead of the exact argument?

```suggestion
      screen.getByText(/i don't know what to say/i),
```

To quote the docs:

> exact: Defaults to true; matches full strings, case-sensitive. When false, matches substrings and is not case-sensitive.
> - exact has no effect on regex or function arguments.
> - In most cases using a regex instead of a string gives you more control over fuzzy matching and should be preferred over { exact: false }.

So, exact isn't doing anything here.